react/features/large-video/components/LargeVideo.web.js

```diff
@@ -5,28 +5,15 @@ import React, { Component } from 'react';
 import { Watermarks } from '../../base/react';
 import { Captions } from '../../subtitles/';
 
-import Labels from './Labels';
-
 declare var interfaceConfig: Object;
 
-/**
- * The type of the React {@code Component} props of {@link LargeVideo}.
- */
-type Props = {
-
-    /**
-     * True if the {@code VideoQualityLabel} should not be displayed.
-     */
-    hideVideoQualityLabel: boolean
-};
-
 /**
  * Implements a React {@link Component} which represents the large video (a.k.a.
  * the conference participant who is on the local stage) on Web/React.
  *
  * @extends Component
  */
-export default class LargeVideo extends Component<Props> {
+export default class LargeVideo extends Component<{}> {
```

Comment:
You can remove the `<{}>`

Reply:
How do you mean?

Reply:
`export default class LargeVideo extends Component {`